app/jenkins-for-jira-ui/src/components/EmptyState/EmptyState.tsx

```diff
@@ -21,10 +25,28 @@ const EmptyStateJenkins = () => {
 	}, []);
 
 	const onClickConnect = () => {
+		analyticsClient.sendAnalytics(
+			AnalyticsEventTypes.UiEvent,
+			AnalyticsUiEventsEnum.ConnectJenkinsServerEmptyStateName,
+			{
+				source: AnalyticsScreenEventsEnum.ConfigurationEmptyStateScreenName,
+				action: 'clickedConnectJenkinsServerEmptyState',
```

Comment:
Can we keep the format for the action consistent? Like `clicked Connect Jenkins server empty state`.